scheduler/rpc/scheduler.go

```diff
@@ -80,11 +77,24 @@ func NewErrors(errs []serror.SnapError) []*SnapError {
 				log.Errorf("Unexpected type %v\n", t)
 			}
 		}
-		errors[i] = &SnapError{ErrorFields: fields, ErrorString: err.Error()}
+		errors[i] = &common.SnapError{ErrorFields: fields, ErrorString: err.Error()}
 	}
 	return errors
 }
 
+func GetError(s *common.SnapError) string {
+	return s.ErrorString
+}
+
+func getFields(s *common.SnapError) map[string]interface{} {
+	fields := make(map[string]interface{}, len(s.ErrorFields))
+	for key, value := range s.ErrorFields {
+		fields[key] = value
+	}
+	return fields
+}
+*/
+
```

Comment:
Definitely. This stuff was moved into common since it was needed in other places.